CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Changelog
 
-## Unreleased
+## 0.3.0.0 (Mar 2023)
 
 * `toChunks` and `toBytes` now make the stdin available to the process being
   executed.
+* Allow streamly 0.9.0
```

Comment:
There are some breaking changes. And deprecations that probably need to be mentioned here.
Also, API changelog can be added.